shared/actions/people.tsx

```diff
@@ -44,7 +44,13 @@ const getPeopleData = async (state: Container.TypedState, action: PeopleGen.GetP
       .reduce(Constants.reduceRPCItemToPeopleItem, I.List())
 
     if (debugTodo) {
-      const allTodos: Array<RPCTypes.HomeScreenTodoType> = Object.values(RPCTypes.HomeScreenTodoType)
```

Comment:
values is actually a mix of the numbers and the strings, so we need to be more explicit